targets/TARGET_NORDIC/TARGET_NRF5/TARGET_SDK11/libraries/util/nrf_assert.h

```diff
@@ -52,6 +52,10 @@
 extern "C" {
 #endif
 
+#if !defined(NDEBUG) && !defined(DEBUG_NRF_USER)
```

Comment:
I'd rather we didn't change the NRF SDK - Could you use the configuration system for these?

Reply:
This is exactly what I tried to start with (above), as @marcuschangarm and I have discussed this was the best I could come up with. 
Especially as I have to change the sdk to fix the variable naming bug in the asserts that Nordic shipped with....